core/src/main/java/org/elasticsearch/indices/IndicesModule.java

```diff
@@ -138,6 +237,15 @@ protected void configure() {
         bind(NodeServicesProvider.class).asEagerSingleton();
     }
 
+    // for testing
+    public synchronized MapperRegistry getMapperRegistry() {
```

Comment:
for the record, this is related to the fact that the base mappers are not hard-coded anymore but use the same registration mechanism as plugins